src/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -687,7 +687,9 @@ A polyhedral fan in ambient dimension 2
 ```
 """
 @attr PolyhedralFan function fan(v::AbstractNormalToricVariety)
-    return PolyhedralFan(pm_object(v))
+    return get_attribute!(v, :fan) do
+        return PolyhedralFan{fmpq}(pm_object(v))
+    end
```

Comment:
```suggestion
    return PolyhedralFan{fmpq}(pm_object(v))
```
The `get_attribute` should not be necessary with the `@attr`, probably comes from a merge-error.